imports/plugins/core/orders/client/templates/workflow/shippingInvoice.js

```diff
@@ -82,6 +83,28 @@ Template.coreOrderShippingInvoice.helpers({
   InvoiceContainer() {
     return InvoiceContainer;
   },
+  buttonSelectComponent() {
+    return {
+      component: ButtonSelect,
+      buttons: [
+        {
+          name: i18next.t("order.approveInvoice"),
```

Comment:
@mikemurray is this how we apply i18n to a button, or should there be an i18nkey/label as props?